src/btree/bt_vrfy.c

```diff
@@ -440,7 +440,8 @@ __verify_tree(
      * well.
      */
     if (__wt_session_prefetch_check(session, ref))
-        WT_RET(__wt_btree_prefetch(session, ref));
+        WT_PREFETCH_ASSERT(
```

Comment:
This is the only instance I can see of `WT_PREFETCH_ASSERT` with side effects. I think the fix here is to say (e.g.)

```
ret = __wt_btree_prefetch(...);
WT_PREFETCH_ASSERT(ret != WT_ERROR);
WT_RET(ret);
```